packages/node/src/schematics/library/library.ts

```diff
@@ -0,0 +1,88 @@
+import { normalize, Path } from '@angular-devkit/core';
+import {
+  apply,
+  chain,
+  externalSchematic,
+  mergeWith,
+  move,
+  Rule,
+  SchematicContext,
+  template,
+  Tree,
+  url,
+  MergeStrategy,
+  noop,
+  filter
+} from '@angular-devkit/schematics';
+import {
+  formatFiles,
+  names,
+  offsetFromRoot,
+  toFileName,
+  updateJsonInTree
+} from '@nrwl/workspace';
+import { Schema } from './schema';
+
+export interface NormalizedSchema extends Schema {
+  name: string;
+  fileName: string;
+  projectRoot: Path;
+  projectDirectory: string;
+  parsedTags: string[];
+}
+
+export default function(schema: NormalizedSchema): Rule {
+  return (host: Tree, context: SchematicContext) => {
+    const options = normalizeOptions(schema);
+
+    return chain([
+      externalSchematic('@nrwl/workspace', 'lib', schema),
+      createFiles(options),
+      formatFiles(options)
+    ])(host, context);
```

Comment:
You can return just the `chain` here. You don't need to call it with `(host, context)`